tests/cli/test_felix.py

```diff
@@ -0,0 +1,125 @@
+import os
+
+import pytest_twisted
+from twisted.internet import threads
+from twisted.internet.task import Clock
+
+from nucypher.blockchain.eth.actors import Miner
+from nucypher.blockchain.eth.token import NU
+from nucypher.characters.chaotic import Felix
+from nucypher.cli import deploy
+from nucypher.cli.main import nucypher_cli
+from nucypher.config.characters import FelixConfiguration
+from nucypher.utilities.sandbox.constants import (
+    TEMPORARY_DOMAIN,
+    TEST_PROVIDER_URI,
+    INSECURE_DEVELOPMENT_PASSWORD,
+    MOCK_CUSTOM_INSTALLATION_PATH_2
+)
+
+
+@pytest_twisted.inlineCallbacks
+def test_run_felix(click_runner, testerchain, federated_ursulas, mock_primary_registry_filepath):
+
```

Comment:
Jeez this is a good test.  Very nice engineering.  I particularly like the clock management and synchronous re-entry to cause the airdrop.  See my comment below about my concerns that this is the only test.